docs/migrating.md

```diff
@@ -50,7 +50,7 @@ When installing `horcrux` we recommend using either the [container image](https:
 The image or binary will be used on each cosigner (bare virtual machine, docker container, kubernetes pod, etc.)
 The binary should also be installed on your local machine for working with the config and key files before distributing to the cosigner nodes.
 
-Run the following on your local machine. If you are using the binary on the cosigners rather than container image, run this on each cosigner node VM also.
+Run the following on your local machine. If you are using the binary on the cosigner rather than container image, run this on each cosigner node VM also.
```

Comment:
guessing this was accidentally changed, same for the rest of the occurrences in this file